src/systemd/sd-login.h

```diff
@@ -191,7 +191,7 @@ int sd_session_get_remote_host(const char *session, char **remote_host);
 int sd_session_get_remote_user(const char *session, char **remote_user);
 
 /* Determine the TTY of this session. */
-int sd_session_get_tty(const char *session, char **display);
+int sd_session_get_tty(const char *session, char **tty);
```

Comment:
if you touch this you might as well rename this `ret_tty`.

our current coding styles dictates that return parameters should be named `ret` or `ret_xyz`, to indicate clearly that they are output-only.

Reply:
hmm, #26590 already addresses this, comprehensively. hence maybe we should just focus on that PR and close this one?

Reply:
Since you made already a change request for the other PR I will close this one.